backend/business-partner-agent/src/main/java/org/hyperledger/bpa/controller/ActivitiesController.java

```diff
@@ -46,12 +49,16 @@ public class ActivitiesController {
     /**
      * List Items, if no filters return all
      *
+     * @param pc         {@link PaginationCommand}
      * @param parameters ActivitySearchParameters Filters for list
      * @return list of {@link ActivityItem}
      */
-    @Get
-    public HttpResponse<List<ActivityItem>> listActivities(@RequestBean @Valid ActivitySearchParameters parameters) {
-        return HttpResponse.ok(activityManager.getItems(parameters));
+    @Get("/{?pc*}")
```

Comment:
api update. did you run update-backend-typings? i do not see any changes of the backend-types.ts

Reply:
Done now. 
Isn't there a step in the pipeline where the update-backend-typings.sh is called?

Reply:
no, but could be done. Currently the files need manually intervention anyway.